Source/Data/Time/KTTimeSeriesFFTW.cc

```diff
@@ -73,6 +73,20 @@ namespace Katydid
         return hist;
     }
 
+    TH1D* KTTimeSeriesFFTW::CreateHistogramImag(const std::string& name) const
```

Comment:
Should the previous function be changed to CreateHistogramReal to maintain consistency?

Reply:
It would provide consistency, but would require more changes upstream in the KTDAC class.  So the benefit to the present approach is that it is a minimal addition that applies to one configuration.  Another possibility would be to add a new function called `CreateHistogramReal()`, but without deleting the `CreateHistogram()` function that is in use by other classes.